api/hypertable/add_reorder_policy.md

```diff
@@ -11,37 +11,35 @@ api:
 
 # add_reorder_policy() <Tag type="community">Community</Tag>
 
-Create a policy to reorder chunks on a given hypertable index in the background.
-You can have only one reorder policy on each hypertable. Only chunks that are the
-third from the most recent are reordered to avoid reordering chunks that are
-still being inserted into. For more information about reordering chunks, see the
-[reorder_chunk][reorder_chunk] section.
+Create a policy to reorder the rows of a hypertable's chunks on a specified index. The policy reorders the rows for all chunks except the two most recent ones, because these are still getting writes. By default, the policy runs every 24 hours. To change the schedule, call [alter_job][alter_job] and adjust `schedule_interval`.
```

Comment:
```suggestion
Create a policy to reorder the rows of a hypertable's chunks on a specific index. The policy reorders the rows for all chunks except the two most recent ones, because these are still getting writes. By default, the policy runs every 24 hours. To change the schedule, call [alter_job][alter_job] and adjust `schedule_interval`.
```